friends/src/Homepage.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+class HomePage extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            name: '',
+            age: '',
+            email: '',
+
+        };
+    }
+    changeHandler = (event) => {
+        this.setState({
+            [event.target.name]: event.target.value
```

Comment:
Remember our zoom: this only works because we have a `name` attribute on our inputs